src/settings.zig

```diff
@@ -188,7 +188,11 @@ pub fn save() void {
 
 pub const launchConfig = struct {
 	pub var cubyzDir: []const u8 = "";
-
+	pub var headlessServer: bool = false;
+	pub var headlessServerWorldName: []const u8 = "";
+	pub var headlessGamemode: main.game.Gamemode = .survival;
+	pub var headlessTestingMode: bool = false;
+	pub var headlessAllowCheats: bool = false;
```

Comment:
Maybe make a generic world config struct, otherwise this implementation here will become out of data quickly without anyone noticing.